dataikuapi/dss/project.py

```diff
@@ -971,6 +971,18 @@ def download_exported_bundle_archive_to_file(self, bundle_id, path):
                     f.flush()
         stream.close()
 
+    def publish_bundle(self, bundle_id, published_project_key=None):
```

Comment:
We don't want to document the fields from the returned object: `PublishedBundleState` ?

Reply:
Done in https://github.com/dataiku/dataiku-api-client-python/commit/8b4fe3041b5b2cc3344b8616da05fa3fe4a114c9 (although the fields are probably not useful from the Python client)